addLocation.html

```diff
@@ -103,15 +101,15 @@
 <body onload="retrieveInformation()">
 
     <h1>Add Location</h1><br>
-    <h3>Metaverse Provider URL</h3>
+    <h3>Location List Provider URL</h3>
     <select id="javascriptURL">
     </select>
     <h3>Domain Name</h3>
-    <input type="text" id="domainName" placeholder="Enter Domain Name here">
+        <input type="text" id="domainName" placeholder="Enter Domain Name here">
```

Comment:
this indentation would indicate that this (ht/x)ml element is in a deeper scope than the one on the previous line. Probably better to use newlines to separate instead

Reply:
All done.